codecov.yml

```diff
@@ -0,0 +1 @@
+comment: false
```

Comment:
want to make same changes as https://github.com/moby/buildkit/pull/1878 for this?

Reply:
I suspect that the issue on buildkit is related to testing on the windows environment (test-os job). I think we can leave it here and see what happens in the future.